editor/src/components/editor/store/dispatch-strategies.spec.tsx

```diff
@@ -1,3 +1,5 @@
+jest.spyOn(Date, 'now').mockReturnValue(new Date(1000).getTime())
```

Comment:
does this interfere with other tests? should this go in a `beforeAll` block (with a corresponding cleanup in `afterAll`)?

Reply:
alternatively, instead of using Date.now() directly in the strategy code, you could make a strategy specific function that gets the date, and mock that. which means all the other places in the codebase that use Date.now() wouldn't be affected by this mock!